tools/ogs-tools/AssignMaterialParameter.xml

```diff
@@ -0,0 +1,50 @@
+<tool id="assign_material_parameter" name="Assign Material Parameter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Assign material parameters to a VTK Unstructured Grid based on a CSV file.
```

Comment:
I would remove `Assign material parameters` here, because the name shown in the Galaxy tool menu will be the name + description (the we have `"Assign material parameter(s)"` twice).